locales/zh_CN/LC_MESSAGES/matrix.po

```diff
@@ -279,155 +306,84 @@ msgstr "（矩阵）该矩阵本身"
 
 #: ../../matrix.rst:132
 msgid ""
-"For example, this extracts the element in row 0 column 1 of matrix ``a``:"
-" ::"
+"For example, this extracts the element in row 0 column 1 of matrix ``a``: ::"
 msgstr "比如，访问矩阵 ``a`` 第0行第1列的元素： ::"
 
 #: ../../matrix.rst:135
 msgid "x = a[0, 1]"
-msgstr ""
+msgstr "x = a[0, 1]"
 
 #: ../../matrix.rst:137
 msgid "This sets the element in row 1 column 3 of ``a`` to 4: ::"
 msgstr "将 ``a`` 第1行第3列的元素设置为4： ::"
 
 #: ../../matrix.rst:140
 msgid "a[1, 3] = 4"
-msgstr ""
+msgstr "a[1, 3] = 4"
 
 #: ../../matrix.rst:143
 msgid "Methods"
-msgstr ""
+msgstr "方法"
 
 #: ../../matrix.rst
 msgid "Returns"
-msgstr ""
+msgstr "返回"
 
 #: ../../matrix.rst:148
-#, fuzzy
 msgid "(Matrix) the transposed matrix of ``a``."
-msgstr "（矩阵）该矩阵本身"
+msgstr "（矩阵） ``a`` 的转置矩阵 。"
 
 #: ../../matrix.rst:150
 msgid "For example::"
-msgstr ""
+msgstr "例如："
 
 #: ../../matrix.rst:152
 msgid ""
 "a = ti.Matrix([[2, 3], [4, 5]])\n"
 "b = a.transpose()\n"
 "# Now b = ti.Matrix([[2, 4], [3, 5]])"
 msgstr ""
+"a = ti.Matrix([[2, 3], [4, 5]])\n"
+"b = a.transpose()\n"
+"# Now b = ti.Matrix([[2, 4], [3, 5]])"
 
 #: ../../matrix.rst:158
 msgid ""
-"``a.transpose()`` will not effect the data in ``a``, it just return the "
-"result."
-msgstr ""
+"``a.transpose()`` will not effect the data in ``a``, it just return the result."
+msgstr "``a.transpose()`` 不会影响 ``a`` 中的数据，它只是返回结果。"
 
 #: ../../matrix.rst:164
-#, fuzzy
 msgid "(scalar) the trace of matrix ``a``."
-msgstr "（标量）矩阵的行数"
+msgstr "（标量） ``a`` 矩阵的痕迹。"
```

Comment:
```suggestion
msgstr "（标量） ``a`` 矩阵的迹。"
```
I thought it's a standard translation in many linalg textbooks... let's follow them.